contents/docs/integrate/server/python.mdx

```diff
@@ -196,10 +199,12 @@ def homepage(request):
 
 If you're using multivariate feature flags, you can also get not just whether the flag is enabled, but what value its enabled to.
 
+> **Note:** Whenever we face an error computing the flag, the library returns `None`, instead of `true` or `false` or a string variant value.
```

Comment:
```suggestion
> **Note:** Whenever we face an error computing the flag, the library returns `None`, instead of `true`, `false`, or a string variant value.
```